src/github-action.js

```diff
@@ -1,10 +1,10 @@
 const core = require('@actions/core');
-const { setup_cml } = require('./utils');
+const { setupCML } = require('./utils');
```

Comment:
Should we use `setupCml` instead of `setupCML` to preserve the naming convention consistency as per what we agreed: https://github.com/iterative/cml/issues/547#issuecomment-845156950

> _By the way: do we agree on the [definition of camel case](https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined)? 😉_

Actually it was a trick question. 😈

Reply:
As per [23.9](https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms) we should use the latter, but... 😬

Reply:
yes we're following ugly e.g. `SshPrivate` elsewhere too